policyengine_us/variables/gov/ed/pell_grant/sai/pell_grant_uses_sai.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+from policyengine_us.variables.gov.ed.pell_grant.pell_grant_calculation_method import (
+    PellGrantCalculationMethod,
+)
+
+
+class pell_grant_uses_sai(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    label = "Pell Grant uses the student aid index"
+
+    def formula(person, period, parameters):
+        method = person.tax_unit("pell_grant_calculation_method", period)
+        return method == PellGrantCalculationMethod.SAI
```

Comment:
```suggestion


class pell_grant_uses_sai(Variable):
    value_type = bool
    entity = Person
    definition_period = YEAR
    label = "Pell Grant uses the student aid index"

    def formula(person, period, parameters):
        method = person.tax_unit("pell_grant_calculation_method", period)
        return method == method.possible_values.PellGrantCalculationMethod.SAI
```

Reply:
I think this is the more conventional way of using the ENUM variables in other classes 
- dont need to import as we use the possible_values function 

I think we can apply this across the PR